node/rpc-client/src/client.rs

```diff
@@ -177,7 +177,15 @@ pub mod filecoin_rpc {
     use jsonrpc_v2::Error;
 
     use crate::call_params;
+<<<<<<< HEAD
+    use rpc_api::{
+        auth_api::*,
+        chain_api::*,
+        mpool_api::{MpoolPendingParams, MpoolPendingResult, MPOOL_PENDING},
+    };
+=======
     use rpc_api::{auth_api::*, chain_api::*, sync_api::*, wallet_api::*};
+>>>>>>> 1b57d550c3363d90f83b2235241fdcce2648ec9d
```

Comment:
👀

Reply:
Weird that this was still able to build while testing. Fixed